includes/sanitizers/class-amp-blacklist-sanitizer.php

```diff
@@ -126,18 +199,25 @@ private function get_blacklisted_tags() {
 			'select',
 			'option',
 			'link',
+			'picture',
+
+			// Sanitizers run after embed handlers, so if anything wasn't matched, it needs to be removed.
+			'embed',
+			'embedvideo',
 
 			// These are converted into amp-* versions
 			//'img',
 			//'video',
 			//'audio',
 			//'iframe',
-		);
+		) );
 	}
 
 	private function get_blacklisted_attributes() {
-		return array(
+		return $this->merge_defaults_with_args( 'add_blacklisted_attributes', array(
 			'style',
-		);
+			'srcset',
```

Comment:
I swear it wasn't supported when I started this, but looks like it was added on 2/11 to the spec. I'll remove this.